pkg/types/types.go

```diff
@@ -19,3 +25,13 @@ type Parser interface {
 	// Parse parses the dependency file
 	Parse(r dio.ReadSeekerAt) ([]Library, []Dependency, error)
 }
+
+type RefType string
+
+const (
+	Website      RefType = "website"
+	License      RefType = "license"
+	VCS          RefType = "vcs"
+	IssueTracker RefType = "issue-tracker"
+	Other        RefType = "other"
```

Comment:
We should have a prefix.

```suggestion
	RefWebsite      RefType = "website"
	RefLicense      RefType = "license"
	RefVCS          RefType = "vcs"
	RefIssueTracker RefType = "issue-tracker"
	RefOther        RefType = "other"
```